changelog/postblitQualifiers.dd

```diff
@@ -21,7 +22,7 @@ void main()
     S a;
     auto a2 = a;
 }
+---
 
-
-The patch makes it so that if a postblit contains const/immutable/shared
-in its signature a deprecation will occur.
+With this release a postblit containing `const`/`immutable`/`shared`
```

Comment:
With this release, if a postblit contains `const`/`immutable`/`shared` its signature, a deprecation message will be emitted.